controllers/dbaas.redhat.com/inventory.go

```diff
@@ -0,0 +1,62 @@
+package dbaasredhatcom
+
+import (
+	dbaasredhatcomv1alpha1 "github.com/CrunchyData/crunchy-bridge-operator/apis/dbaas.redhat.com/v1alpha1"
+	"github.com/CrunchyData/crunchy-bridge-operator/internal/bridgeapi"
+	dbaasv1alpha1 "github.com/RHEcosystemAppEng/dbaas-operator/api/v1alpha1"
+	"github.com/go-logr/logr"
+	"strconv"
+)
+
+const (
+	TEAM_ID                = "team_id"
+	PROVIDER_ID            = "provider_id"
+	REGION_ID              = "region_id"
+	CREATED_AT             = "created_at"
+	UPDATED_AT             = "updated_at"
+	MAJOR_VERSION          = "major_version"
+	STORAGE                = "storage"
+	CPU                    = "cpu"
+	MEMORY                 = "memory"
+	IS_HA                  = "is_ha"
+	CRUNCHYBRIDGE_POSTGRES = "CrunchyBridge Postgres"
```

Comment:
`CrunchyBridge Postgres` is not an identifier we would like to use anywhere. To be consistent with the bundle metadata, I believe this would be `Crunchy Bridge managed PostgreSQL`

Reply:
updated